components/BottomNavigation/tests/unit/BottomNavigationItemViewTests.m

```diff
@@ -35,64 +35,11 @@ @interface MDCBottomNavigationItemView (Testing)
 @property(nonatomic, strong) UILabel *label;
 @end
 
-@interface BottomNavigationItemViewCodingTests : XCTestCase
+@interface BottomNavigationItemViewTests : XCTestCase
 
 @end
```

Comment:
Is this an extra `@end`?

Reply:
Nope - it's closing the BottomNavigationItemViewTests interface declaration.